fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -331,6 +331,7 @@ public class SessionVariable implements Serializable, Writable, Cloneable {
 
     public static final String QUERY_CACHE_AGG_CARDINALITY_LIMIT = "query_cache_agg_cardinality_limit";
     public static final String TRANSMISSION_ENCODE_LEVEL = "transmission_encode_level";
+    public static final String RPC_HTTP_MIN_SIZE = "rpc_http_min_size";
```

Comment:
It should be upper_size instead of lower_size?
`PRC_PACKET_THRESHOLD_SIZE`

Reply:
if the real packet's size > `rpc_http_min_size`, the rpc should be via http. it's also the max size of the std rpc.